rpc_jobs/standard_job_premerge.yml

```diff
@@ -11,10 +11,10 @@
     image:
       - "xenial"
     scenario:
-      - "functional"
       - "lint"
     action:
-      - "test"
+      - "test":
+          SLAVE_TYPE: "nodepool-ubuntu-xenial"
```

Comment:
Would it be cleaner to place this under the image axis so it directly corresponds to the xenial image?  That way if we add another image we don't have to move it later.

Reply:
Sure, can do that if you like.

Reply:
Done.